src/Form.tsx

```diff
@@ -3,14 +3,17 @@ import * as React from "react";
 import { Field, FieldProps } from "formik";
 import { Form as $Form } from "antd";
 import { FormProps } from "antd/lib/form";
-import { FormItem } from './form-item';
+import { FormItem } from "./form-item";
 
-export function Form(props: FormProps) {
-  return (<Field>
-    {({ form: { handleReset, handleSubmit } }: FieldProps) => (
-      <$Form onReset={handleReset} onSubmit={handleSubmit} {...props} />
-    )}
-  </Field>)
-};
+export const Form = (props: FormProps) =>
+  name ? (
```

Comment:
`name` here should come from the props object in line 8. So `props.name` or destructuring `props`